Traceback (most recent call last):
  File "pk.py", line 115, in <module>
    main(args.student1_id, args.student2_id)
  File "pk.py", line 91, in main
    game_results = play_games(student1_id, student1_ai, student2_id, student2_ai)
  File "pk.py", line 57, in play_games
    config.register_player(name=student2_id, algorithm=student2_ai())
  File "/home/ubitus03/fai_final_project/./b11705043/agent.py", line 298, in setup_ai
    return GBDTAgent()
  File "/home/ubitus03/fai_final_project/./b11705043/agent.py", line 16, in __init__
    self.load_training_data()  # Load training data from file
  File "/home/ubitus03/fai_final_project/./b11705043/agent.py", line 39, in load_training_data
    self.training_data = joblib.load(f)
  File "/home/ubitus03/miniconda3/envs/poker/lib/python3.8/site-packages/joblib/numpy_pickle.py", line 648, in load
    obj = _unpickle(fobj)
  File "/home/ubitus03/miniconda3/envs/poker/lib/python3.8/site-packages/joblib/numpy_pickle.py", line 577, in _unpickle
    obj = unpickler.load()
  File "/home/ubitus03/miniconda3/envs/poker/lib/python3.8/pickle.py", line 1210, in load
    raise EOFError
EOFError
